src/sidebars/favourites/tearout.js

```diff
@@ -167,11 +167,9 @@
                                                 dragService = null;
                                             } else {
                                                 // Create new window instance
-                                                var mainApplicationWindowPosition = geometryService.getWindowPosition(window);
-
                                                 var compact = store.isCompact();
                                                 windowService.createMainWindow(null, compact, (newWindow) => {
-                                                    newWindow.resizeTo(mainApplicationWindowPosition.width, mainApplicationWindowPosition.height, 'top-left');
+                                                    newWindow.resizeTo(window.outerWidth, window.outerHeight, 'top-left');
```

Comment:
Yes, just unravelling the pointless call to the geometry service, the code is identical though.